api/proto/teleport/legacy/types/types.proto

```diff
@@ -3416,6 +3420,18 @@ message RoleConditions {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "github_permissions,omitempty"
   ];
+
+  // AccountAssignmentLabels is the label set used to match any sort of
+  // Identity Center resource
+  wrappers.LabelValues AccountAssignmentLabels = 44 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "account_assignment_labels,omitempty",
```

Comment:
Two concerns on label name `account_assignment_labels`:
1. This label will be obsolete/ or we need to introduce another label once again when we support requesting access to both AWS account and AWS application.
2. `account_assignment_labels` is generic enough and might collide labels we might need for future integrations such as GCP accounts, Azure accounts etc (if we ever end up implementing them).  

Can we get away with `aws_ic_assigment_labels` that can be applied to both AWS account and application?